frontend/src/modules/EditZing/Components/Helpers.tsx

```diff
@@ -0,0 +1,47 @@
+import { FetchedZing } from 'EditZing/Types/Student'
+const axios = require('axios')
+
+export async function getZingGroups(docId: String): Promise<FetchedZing> {
+  return fetch(
+    `https://us-central1-zing-backend.cloudfunctions.net/api/course/${docId}`
```

Comment:
ill be replacing the API url with constants that i define in Routes.ts. I can deal with that.

Reply:
ok i wont change anything